src/webview/static/base.css

```diff
@@ -564,3 +564,71 @@ hr {
   border: 1px dashed hsla(0, 0%, 50%, 0.5);
   border-radius: 0.5rem;
 }
+.spoiler-block {
```

Comment:
This chunk of styles would benefit from a comment saying something much like what's in the commit message: it's taken from the `.spoiler-block` section of the webapp's styles, as of such-and-such a commit, just translated from SCSS to plain CSS.